Controller/CoinsController.cs

```diff
@@ -35,18 +33,21 @@ public void EarnCoins(object sender, EventArgs e)
             if(counter == 1000)
             {
                 counter = 0;
-                AddCoins(UserController.CurrentUser);
+                AddCoins();
             } 
         }
 
-        public void AddCoins(User user, int coins = 1)
```

Comment:
En waarom is hier die `User` uit?

Reply:
We kregen een foutmelding als user gebruikt werd, was iets met dat het niet getracked kon worden.

Reply:
Wut?

Reply:
![image](https://user-images.githubusercontent.com/46564984/101176023-32a1b080-3646-11eb-8edd-9c824b972687.png)

Reply:
Nou problem solved, ander UserID gebruiken... xD

Reply:
dus als gebruiker 1 een song luistered krijgt een andere user de coins?

Reply:
Nee? Maar er gaat blijkbaar iets fout. Dat er een GetItem() voor dat userid gedaan wordt, en dat gebeurd ergens nog een keer.

Reply:
kom ns onze call joinen, ik kan je precies laten zien wat er gebeurd en hoe we het opgelost hebben

Reply:
kom **schaken**